.buildkite/scripts/steps/es_serverless/promote_es_serverless_image.sh

```diff
@@ -19,7 +19,12 @@ else
   SOURCE_IMAGE="$BASE_ES_SERVERLESS_REPO:$SOURCE_IMAGE_OR_TAG"
 fi
 
-echo "--- Promoting ${SOURCE_IMAGE_OR_TAG} to ':latest-verified'"
+if [[ "${SKIP_PROMOTION:-}" =~ ^(1|true)$ ]]; then
```

Comment:
Maybe we should invert this, and use explicit request for promotion? That will probably reduce the chance of misusing by 3rd parties.

Reply:
I agree, your inverted approach with `PUBLISH_DOCKER_TAG` probably comes with less surprises. I'm sure you have that on your list anyway, just mentioning that with this, we also need to update our verify&promote trigger (I think [this](https://github.com/elastic/kibana-buildkite/blob/395259f9a4b45e49797c6bfbfce28fff1ac84f88/pipelines/kibana-es-snapshots.tf#L117) one) to actually set the `PUBLISH_DOCKER_TAG` env variable.

Reply:
yes, thanks for reminding!